app/renderer/src/store/settings/defaultSettings.ts

```diff
@@ -17,4 +17,5 @@ export const defaultSettings: Readonly<SettingTypes> = Object.freeze({
   autoStartWorkTime: false,
   useNativeTitlebar: detectOS() === "Windows" ? false : true,
   openAtLogin: false,
+  showTimerInTray: detectOS() === "MacOS",
```

Comment:
I need to take a look as I haven't open the code for this in a while but you'll also need to check if this is the electron version as to if you show this. Otherswise you could also add the rust code to implement it on the tauri side.